### PR TITLE
Add Missing Descriptions

### PR DESCRIPTION
**Pull request by Lifeismana**
### Brief explanation of the change
Added missing descriptions
For now it's a draft until i finish doing missing ones and maybe change the ones i really don't like.

### CODE REVIEW

**Comment by Lifeismana** [comment]
This is it, it has every missing description

**Comment by Lifeismana** [comment]
Or should i add descriptions for Evidence ?

**Comment by xPaw** [comment]
Site won't use them, but it might be ok for documentation purposes.

**Comment by xPaw** [comment]
`**BIF File** any file with the .bif extension.` is just as good as having no description, haha.

Just skip evidence for now.

**Comment by Lifeismana** [comment]
That's what i though, haha

**Comment by larsiusprime** [comment]
A few evidence files I did give descriptions that say things like "this can be a sign of XYZ engine" but yeah let's skip evidence for now.

You have an "is is" typo in one of these, but they all look pretty good to me.

File: descriptions/Engine.ChromeEngine.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[**Chrome Engine**](https://web.archive.org/web/20170506215137/https://technology.techland.pl/) is a proprietary 3D game engine developed by Techland.
```
Comment (xPaw) [inline descriptions/Engine.ChromeEngine.md:1]: https://en.wikipedia.org/wiki/Chrome_Engine is a better link I think.

File: descriptions/Engine.Phyre.md [hunk 2]
```diff
@@ -0,0 +1 @@
+**PhyreEngine** is a free to use, cross platform (Windows PC, PSP, Xbox 360, PS Vita and PS3) game engine from Sony Computer Entertainment.
```
Comment (xPaw) [inline descriptions/Engine.Phyre.md:1]: https://en.wikipedia.org/wiki/PhyreEngine

File: descriptions/Engine.RenPy.md [hunk 3]
```diff
@@ -0,0 +1 @@
+[Ren'Py](https://renpy.org/) is a free visual novel engine running on Python.
```
Comment (xPaw) [inline descriptions/Engine.RenPy.md:1]:
```suggestion
[**Ren'Py**](https://renpy.org/) is a free visual novel engine running on Python.
```

File: descriptions/SDK.NodeJS.md [hunk 4]
```diff
@@ -0,0 +1 @@
+[**NodeJs**] is an open-source, cross-platform, back-end JavaScript runtime environment that runs on the V8 engine and executes JavaScript code outside a web browser.
```
Comment (xPaw) [inline descriptions/SDK.NodeJS.md:1]:
```suggestion
[**Node.js**](https://nodejs.org/) is an open-source, cross-platform, back-end JavaScript runtime environment that runs on the V8 engine and executes JavaScript code outside a web browser.
```

File: descriptions/Engine.idTech2.md [hunk 5]
```diff
@@ -0,0 +1 @@
+**id Tech 2** is a proprietary game engine developed by id Software. It has 2 variants, the Quake engine and the Quake II engine respectively used in [Quake](https://steamdb.info/app/2310/) and in [Quake II](https://steamdb.info/app/2320/)
```
Comment (larsiusprime) [inline descriptions/Engine.idTech2.md:1]: We have a separate classification for Quake Engine already.

We should probably rename Quake->idTech2, and rename idTech2 to idTech2.5 and clarify that idTech2 is "the Quake Engine" and idTech2.5 is "the Quake II Engine." I've seen this on various wikis and it seems the most consistent way to go.

File: descriptions/Engine.Lime_OR_OpenFL.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+[**Lime**](https://lime.software/) is a flexible, lightweight layer for Haxe cross-platform developers. Lime provides a common foundation on native and dynamic targets for window and application life-cycle management, access to rendering and sound, plus many other useful features for cross-platform development.
+
+[**OpenFl**](https://www.openfl.org/) is a free and open-source software framework and platform for the creation of multi-platform applications and video games. OpenFL applications can be written in Haxe, JavaScript or TypeScript.
```
Comment (xPaw) [inline descriptions/Engine.Lime_OR_OpenFL.md:3]: It might be beneficial to say why we group them?

Comment (larsiusprime) [inline descriptions/Engine.Lime_OR_OpenFL.md:3]: Should specify that OpenFL is necessarily built on lime and we are currently unable to distinguish between them, which is why they're combined in one rule.

File: descriptions/Engine.idTech2.md (hunk 5 above)
Comment (Lifeismana) [reply descriptions/Engine.idTech2.md:1]: or we could use a weird name like id Tech 2 Quake Edition and id Tech 2 Quake II Edition 
(this is probably worse)

**Comment by xPaw** [comment]
I merged this, the rest can be iterated in a future pr.

**Pull request merged**